The FreeDesktop icon themes specifications tells to put the icons in a theme
subdirectory in /usr/share/icons, and to use the name of the icon without
extension in the .desktop files, to allow themes to use versions in other
formats. For the same reason, gxine-logo.png icon is installed as gxine.png,
and just gxine is used in the .desktop file.
Index: gxine-0.5.0/gxine.desktop
===================================================================
--- gxine-0.5.0.orig/gxine.desktop
+++ gxine-0.5.0/gxine.desktop
@@ -6,7 +6,7 @@ GenericName=Video Player
 Comment=Play films and songs, or watch digital TV
 Exec=gxine
 MultipleArgs=true
-Icon=gxine-logo.png
+Icon=gxine
 Terminal=false
 Type=Application
 StartupNotify=true
Index: gxine-0.5.0/src/desktop_integration.c
===================================================================
--- gxine-0.5.0.orig/src/desktop_integration.c
+++ gxine-0.5.0/src/desktop_integration.c
@@ -349,7 +349,7 @@ void di_register_kde (void) {
 	   "Comment=Play films and songs, or watch digital TV\n"
 	   "MultipleArgs=true\n"
 	   "Terminal=false\n"
-	   "Icon=%s/gxine-logo.png\n"
+	   "Icon=gxine\n"
 	   "Exec=%s/gxine\n"
 	   "Type=Application\n"
 	   "StartupNotify=true\n"
Index: gxine-0.5.0/src/ui.c
===================================================================
--- gxine-0.5.0.orig/src/ui.c
+++ gxine-0.5.0/src/ui.c
@@ -779,8 +779,8 @@ static void ui_load_icons (void)
 
   ui_std_icon (ifactory, GXINE_MEDIA_SPEAKER, "speaker.png");
   ui_std_icon (ifactory, GXINE_MEDIA_SPEAKER_MUTE, "nospeaker.png");
-  ui_std_icon (ifactory, GXINE_MEDIA_MARK, "/gxine-logo.png");
-  ui_std_icon (ifactory, GXINE_LOGO, "/gxine-logo.png");
+  ui_std_icon (ifactory, GXINE_MEDIA_MARK, "/gxine.png");
+  ui_std_icon (ifactory, GXINE_LOGO, "/gxine.png");
 
   gtk_icon_factory_add_default (ifactory);
 }
